daemon/containerd/image_prune.go

```diff
@@ -229,15 +228,15 @@ func (i *ImageService) pruneAll(ctx context.Context, imagesToPrune map[string]c8
 			return nil
 		})
 		if err != nil {
-			errs = multierror.Append(errs, err)
+			errs = errors.Join(errs, err)
 			if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
 				return &report, errs
 			}
```

Comment:
To get the same formatting as before, should `errs` be declares as `var errs []error`, then ...

```suggestion
			errs = append(errs, err)
			if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
				return &report, return errors.Join(errs...)
			}
```

And the same in the next block, with the return at the end of the function being `return &report, errors.Join(errs...)`